source-azure-blob-storage/.snapshots/TestAzureBlobStore_getConfigSchema

```diff
@@ -0,0 +1,110 @@
+{
+		"$schema": "http://json-schema.org/draft-07/schema#",
+		"title": "Azure Blob Storage Source",
+		"type": "object",
+		"properties": {
+			"credentials": {
+				"type": "object",
+				"title": "Credentials",
+				"description": "Azure credentials used to authenticate with Azure Blob Storage.",
+				"order": 0,
+				"anyOf": [
+					{
+						"title": "OAuth2 Credentials",
```

Comment:
Ah so is this really an OAuth2 flow kind of authentication? If so we'd probably want to get an OAuth app approved and have it work via our UI. If that's the case, I'm thinking it isn't worth the effort right now and we should only support the "connection string" auth, which is TBH the only one of the auth methods I could figure out myself. We'd still want this to be a `oneOf` (you've got it as `anyOf` right now, which I think we should change before merging to be like [this](https://github.com/estuary/connectors/blob/6ec57528da06bec6846c307fc7b48c651e2ac9c6/source-sftp/main.go#L456)) so that we could expand it in the future by adding to the list, even though there'd only be one option.

Reply:
About the OAuth2 authentication, we don't need to create an app by ourselves. The customer needs to. This is the recommended way in the docs, you can see how to set up it and more information on the docs PR here: https://github.com/estuary/flow/pull/1441/
https://github.com/estuary/flow/blob/1c2b31df606ce41956c9e848ed6d5506b749251e/site/docs/reference/Connectors/capture-connectors/azure-blob-storage.md

Reply:
Oh I see, thanks! 

Should we make this a `oneOf`? I think we should. since technically it would not be value for a config to match more than one of the options.